<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>MainWindow</class>
  <widget class="QMainWindow" name="MainWindow">
    <property name="geometry">
      <rect>
	<x>0</x>
	<y>0</y>
	<width>400</width>
	<height>432</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>MainWindow</string>
    </property>
    <widget class="QWidget" name="centralWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
	<item>
	  <widget class="QGroupBox" name="gbSettings">
	    <property name="title">
	      <string>Settings</string>
	    </property>
	    <layout class="QFormLayout" name="formLayout">
	      <item row="0" column="1">
		<widget class="QWidget" name="widget2" native="true">
		  <layout class="QGridLayout" name="gridLayout_3">
		    <property name="leftMargin">
		      <number>0</number>
		    </property>
		    <property name="topMargin">
		      <number>0</number>
		    </property>
		    <property name="rightMargin">
		      <number>0</number>
		    </property>
		    <property name="bottomMargin">
		      <number>0</number>
		    </property>
		    <property name="horizontalSpacing">
		      <number>0</number>
		    </property>
		    <item row="0" column="0">
		      <widget class="QLineEdit" name="txtMCPEWorld">
			<property name="toolTip">
			  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The MCPE World that you want to use&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
			</property>
		      </widget>
		    </item>
		    <item row="0" column="1">
		      <widget class="QToolButton" name="btnMCPEWorld">
			<property name="text">
			  <string>...</string>
			</property>
		      </widget>
		    </item>
		  </layout>
		</widget>
	      </item>
	      <item row="1" column="0">
		<widget class="QLabel" name="label_4">
		  <property name="text">
		    <string>Output Directory</string>
		  </property>
		</widget>
	      </item>
	      <item row="1" column="1">
		<widget class="QWidget" name="widget1" native="true">
		  <layout class="QGridLayout" name="gridLayout_2">
		    <property name="leftMargin">
		      <number>0</number>
		    </property>
		    <property name="topMargin">
		      <number>0</number>
		    </property>
		    <property name="rightMargin">
		      <number>0</number>
		    </property>
		    <property name="bottomMargin">
		      <number>0</number>
		    </property>
		    <property name="spacing">
		      <number>0</number>
		    </property>
		    <item row="0" column="0">
		      <widget class="QLineEdit" name="txtOutputDirectory">
			<property name="toolTip">
			  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Your output images / web app will be in this directory&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
			</property>
			<property name="text">
			  <string/>
			</property>
		      </widget>
		    </item>
		    <item row="0" column="1">
		      <widget class="QToolButton" name="btnOutputDirectory">
			<property name="text">
			  <string>...</string>
			</property>
		      </widget>
		    </item>
		  </layout>
		</widget>
	      </item>
	      <item row="2" column="0">
		<widget class="QLabel" name="label_2">
		  <property name="text">
		    <string>Output Name</string>
		  </property>
		</widget>
	      </item>
	      <item row="2" column="1">
		<widget class="QLineEdit" name="txtOutputName">
		  <property name="toolTip">
		    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Your output filenames will start with this (e.g. &amp;quot;out1&amp;quot;)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
		  </property>
		  <property name="text">
		    <string/>
		  </property>
		</widget>
	      </item>
	      <item row="0" column="0">
		<widget class="QLabel" name="label_3">
		  <property name="text">
		    <string>MCPE World</string>
		  </property>
		</widget>
	      </item>
	      <item row="3" column="0">
		<widget class="QLabel" name="label">
		  <property name="text">
		    <string>Output Mode</string>
		  </property>
		</widget>
	      </item>
	      <item row="3" column="1">
		<widget class="QComboBox" name="cbImages">
		  <property name="toolTip">
		    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select the type of data you want&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
		  </property>
		  <item>
		    <property name="text">
		      <string comment="1">Web App with Most Images</string>
		    </property>
		  </item>
		  <item>
		    <property name="text">
		      <string comment="2">Web App with ALL Images</string>
		    </property>
		  </item>
		  <item>
		    <property name="text">
		      <string comment="3">Most Images</string>
		    </property>
		  </item>
		  <item>
		    <property name="text">
		      <string comment="4">All Images</string>
		    </property>
		  </item>
		</widget>
	      </item>
	      <item row="4" column="0">
		<widget class="QLabel" name="label_5">
		  <property name="text">
		    <string>Command Line</string>
		  </property>
		</widget>
	      </item>
	      <item row="4" column="1">
		<widget class="QLineEdit" name="txtCommandLine">
		  <property name="toolTip">
		    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This will contain the command-line used to run mcpe_viz.  If you like, you can copy it and use it in a command prompt to customize the way mcpe_viz runs.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
		  </property>
		  <property name="text">
		    <string/>
		  </property>
		  <property name="readOnly">
		    <bool>true</bool>
		  </property>
		</widget>
	      </item>
	    </layout>
	  </widget>
	</item>
	<item>
	  <widget class="QPushButton" name="btnGo">
	    <property name="toolTip">
	      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Run MCPE Viz!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
	    </property>
	    <property name="text">
	      <string>Go!</string>
	    </property>
	  </widget>
	</item>
	<item>
	  <widget class="QGroupBox" name="gbProgress">
	    <property name="title">
	      <string>Progress</string>
	    </property>
	    <layout class="QHBoxLayout" name="horizontalLayout">
	      <property name="leftMargin">
		<number>2</number>
	      </property>
	      <property name="topMargin">
		<number>2</number>
	      </property>
	      <property name="rightMargin">
		<number>2</number>
	      </property>
	      <property name="bottomMargin">
		<number>2</number>
	      </property>
	      <item>
		<widget class="QPlainTextEdit" name="txtProgress">
		  <property name="toolTip">
		    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;After you press &amp;quot;Go!&amp;quot;, you will see the progress of MCPE Viz here.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
		  </property>
		  <property name="lineWrapMode">
		    <enum>QPlainTextEdit::NoWrap</enum>
		  </property>
		  <property name="readOnly">
		    <bool>true</bool>
		  </property>
		</widget>
	      </item>
	    </layout>
	    <zorder>txtProgress</zorder>
	    <zorder>gbSettings</zorder>
	  </widget>
	</item>
	<item>
	  <widget class="QPushButton" name="btnWebApp">
	    <property name="toolTip">
	      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Once MCPE Viz has run, you can press this button to launch the web app!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
	    </property>
	    <property name="text">
	      <string>Launch Web App</string>
	    </property>
	  </widget>
	</item>
	<item>
	  <widget class="QPushButton" name="btnCheckUpdate">
	    <property name="toolTip">
	      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Click this to check for MCPE Viz updates on GitHub.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
	    </property>
	    <property name="text">
	      <string>Check for MCPE Viz Update</string>
	    </property>
	  </widget>
	</item>
      </layout>
    </widget>
    <widget class="QStatusBar" name="statusBar"/>
    <action name="action_About">
      <property name="text">
	<string>&amp;About</string>
      </property>
    </action>
  </widget>
  <layoutdefault spacing="6" margin="11"/>
  <tabstops>
    <tabstop>txtMCPEWorld</tabstop>
    <tabstop>btnMCPEWorld</tabstop>
    <tabstop>txtOutputDirectory</tabstop>
    <tabstop>btnOutputDirectory</tabstop>
    <tabstop>txtOutputName</tabstop>
    <tabstop>cbImages</tabstop>
    <tabstop>txtCommandLine</tabstop>
    <tabstop>btnGo</tabstop>
    <tabstop>txtProgress</tabstop>
    <tabstop>btnWebApp</tabstop>
    <tabstop>btnCheckUpdate</tabstop>
  </tabstops>
  <resources/>
  <connections/>
</ui>
